digitalpublicgoods/synthea.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Synthea",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/synthetichealth/synthea#license"
```

Comment:
Is this submission attributed to the MITRE Corporation?

Reply:
Clarified [here](https://github.com/DPGAlliance/publicgoods-candidates/pull/1037/files#diff-350268f0100a8dcf3c2eeb72f619f7d153d2c2b5cc2c34503f64528f818a55e8R43) in PR #1037.